cmd/wallet-sdk-gomobile/docs/usage.md

```diff
@@ -347,7 +344,6 @@ interaction.authorize() // Returned object doesn't matter with current implement
 val userPIN = "1234"
 val requestCredentialOpts = CredentialRequestOpts(userPIN)
 val credentials = interaction.requestCredential(requestCredentialOpts, doc.assertionMethod()) // Should probably store these somewhere
```

Comment:
While we're at it, I noticed that this line refers to `doc` while line 337 above sets `didDocResolution`.

Reply:
Good catch - fixed.